deps.sh

```diff
@@ -0,0 +1,26 @@
+# when specifying a commit that is not on the main repository yet but in a PR,
+# you'll have to specify the GitHUB PR number so that we can fetch it 
+
+export RPMS_COMMIT=
+export RPMS_PR=97
```

Comment:
as you can see I'm referencing unmerged PRs here; these are the draft PRs that contain the changes necessary to make all this magically work